.github/workflows/perftest-server.yml

```diff
@@ -0,0 +1,168 @@
+name: Prepare Artipie for perftests
+concurrency: perftest_env
+on:
+  push:
+    tags:
+      - "t*"
+      - "v*"
+
+jobs:
+  perf-test1:
+    runs-on: [wsl-server]
+    steps:
+      - uses: actions/checkout@v3
+      - run: cat /etc/issue; mount; ls -lah; env; pwd; git status
+      - name: Check variables
+        run: test ! -f $HOME/.benchvars && exit 1;exit 0
+      - name: Prepare docker env
+        run: |
+          docker info
+          docker stop artipie || :
+          docker rm -fv artipie || :
+          docker image rm -f artipie/artipie:1.0-SNAPSHOT || :
+          docker ps
+      - name: Maven build
+        run: timeout 600 mvn clean install
```

Comment:
I'd suggest skip tests and use docker profile
`mvn clean install -Pdocker -DskipTests`

Reply:
@olenagerasimova , thanks.